scripts/conversion_script.py

```diff
@@ -430,6 +460,15 @@ def main():
     ]
 
     try:
+        # Exit if not CentOS 7.9
+        dist, version = get_system_distro_version()
```

Comment:
the script will fail if the IOError occurs inside the funcion, the unpack won't work, are we fine with that or do we just want to continue the script?

Reply:
ugh I don't know what I meant by this (or if it was fixed in the meantime), but It should return `None, None` in case of error right? maybe I just didn't see it when it was inside the try

Reply:
> ugh I don't know what I meant by this (or if it was fixed in the meantime), but It should return `None, None` in case of error right? maybe I just didn't see it when it was inside the try

Yeah, that's correct.